elife/templates/systemd-multiple-processes.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+NUMBER={{ number }}
+timeout=120
+    echo "--------"
```

Comment:
not sure there is a reason anymore for the additional indentation?

Reply:
quite right, I think I removed some enclosing conditional